Description: remove feature gstreamer_backend
 ffmpeg_gst_wrapper is built without the feature
 ffmpeg_gst_wrapper/gstreamer_backend anyway
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,10 +36,6 @@
 app_only_fns = []
 debug_hash_generation = []
 default = []
-gstreamer_backend = [
-    "gstreamer",
-    "ffmpeg_gst_wrapper/gstreamer_backend",
-]
 test-util = []
 
 [lib]
@@ -96,7 +92,3 @@
 
 [dev-dependencies.itertools]
 version = "0.14"
-
-[target."cfg(unix)".dependencies.gstreamer]
-version = "0.23"
-optional = true
